spec-0007/index.md

```diff
@@ -97,126 +97,169 @@ The [deprecation strategy](https://github.com/scientific-python/specs/pull/180#i
 
    ```
 
-   Also note the suggested language for the `rng` parameter docstring.
+   Also note the suggested language for the `rng` parameter docstring, which encourages the user to pass a `Generator` or None, but allows for other types accepted by `numpy.random.default_rng`.
 
 ### Impact
 
-The following users will be affected:
+There are three classes of users, which will be affected to varying degrees.
 
-1. Those who use `np.random.seed`.
-   The proposal will do away with that global seeding mechanism, meaning that code that relies on it will, after a certain deprecation period, start seeing a different stream of random numbers than before.
-   To ensure that this does not go unnoticed, the library should raise a `FutureWarning` if `np.random.seed` was called earlier (we show how to do that further down).
+1. Those who do not attempt to control the random state.
+   Their code will immediately switch from using the unseeded global `RandomState` to using an unseeded `Generator`.
+   Since the underlying _distributions_ of pseudo-random numbers will not change, these users will be relatively unaffected.
```

Comment:
I wonder about that word "relatively". Maybe "should be unaffected"?

Reply:
OK.

Reply:
Done.